### PR TITLE
170 change translations

### PR DESCRIPTION
**Pull request by Ssravani272**
- Change the web text for English and Bahasa language for the training platform

### CODE REVIEW

File: deployment_specific/pb/ds_locales/en.js [hunk 1]
```diff
@@ -171,63 +171,58 @@ const en = {
       'title': 'Watch Instructional Video'
     },
     'whatsapp': {
-      'title': 'Report via WhatsApp',
+      'title': 'Complete Training Exercise via WhatsApp',
       'steps': [
-        'Send a message to +62858-4-BENCANA (+62 858-4-2362262) or <a target="_blank" href="https://bit.ly/BencanaBotWA">https://bit.ly/BencanaBotWA</a>',
+        'Send a message to +62858-4-BENCANA (+62 858-4-2362262) or https://bit.ly/BencanaBotWA',
```
Comment (ruthvik129) [inline deployment_specific/pb/ds_locales/en.js:176]: Make sure to have the link in a href with target blank

File: deployment_specific/pb/ds_locales/en.js [hunk 2]
```diff
@@ -171,63 +171,58 @@ const en = {
...
+        'Open the link, select “complete training exercise”, and follow the instructions to input hazard location, upload photo and description.',
+        'DisasterBot will send you a reply with a link to your simulation report.'
       ]
     },
     'twitter': {
-      'title': 'Report Via Twitter',
+      'title': 'Complete Training Exercise via X (formerly Twitter)',
       'steps': [
-        'Tweet <a href="https://twitter.com/intent/tweet?text=@petabencana">@petabencana</a> with one of the following hashtags:<ul><li>#flood</li><li>#earthquake</li><li>#forestfire</li><li>#haze</li><li>#volcano</li></ul>',
+        'Tweet @petabencana with one of the following hashtags:<ul><li>#flood</li><li>#earthquake</li><li>#forestfire</li><li>#haze</li><li>#volcano</li></ul>',
```
Comment (ruthvik129) [inline deployment_specific/pb/ds_locales/en.js:185]: Make sure to have petabencana tag to be in ahref look at the change in red it should be like that

File: deployment_specific/pb/ds_locales/id.js [hunk 3]
```diff
@@ -171,74 +171,69 @@ const id = {
...
     'whatsapp': {
-      'title': 'Laporkan melalui WhatsApp',
+      'title': 'Selesaikan simulasi melalui WhatsApp',
       'steps': [
-        'Kirim pesan ke +62858-4-BENCANA (+62 858-4-2362262) atau  <a href="https://bit.ly/BencanaBotWA">https://bit.ly/BencanaBotWA</a>',
-        'BencanaBot akan mengirimkan tautan untuk melaporkan bencana.',
-        'Buka tautan tersebut dan ikuti petunjuk untuk melaporkan lokasi, unggah foto dan deskripsi.',
-        'BencanaBot akan mengirim balasan dengan tautan yang akan mengarahkan ke laporan Anda.',
-        'Cek laman PetaBencana.id untuk melihat informasi bencana terkini.'
+        'Kirim pesan ke +62858-4-BENCANA (+62 858-4-2362262) atau https://bit.ly/BencanaBotWA',
```
Comment (ruthvik129) [inline deployment_specific/pb/ds_locales/id.js:176]: Do the same as english version , use a tags

File: deployment_specific/pb/ds_locales/id.js [hunk 4]
```diff
@@ -171,74 +171,69 @@ const id = {
...
     'twitter': {
-      'title': 'Laporkan melalui Twitter',
+      'title': 'Selesaikan simulasi melalui X (sebelumnya bernama Twitter)',
       'steps': [
-        'Twit <a href="https://twitter.com/intent/tweet?text=@petabencana">@petabencana</a> dengan salah satu hashtag berikut:<ul><li>#banjir</li><li>#gempa</li><li>#kebakaranhutan</li><li>#kabutasap</li><li>#gunungapi</li></ul>',
-        '@petabencana akan mengirimkan tautan untuk melaporkan bencana',
-        ' Buka tautan tersebut dan ikuti petunjuk untuk melaporkan lokasi, unggah foto dan deskripsi',
-        '@petabencana akan mengirim balasan twit dengan tautan yang akan mengarahkan ke laporan Anda',
-        'Cek laman PetaBencana.id untuk melihat informasi bencana terkini'
+        'Buat cuitan ke @petabencana dengan salah satu tagar berikut:<ul><li>#Banjir</li><li>#Gempabumi</li><li>#Kebakaranhutan</li><li>#Kabutasap</li><li>#Gunungberapi</li></ul>',
```
Comment (ruthvik129) [inline deployment_specific/pb/ds_locales/id.js:185]: Do the same as english version , use a tags

**Review by ruthvik129 — CHANGES_REQUESTED** [review]

File: deployment_specific/pb/ds_locales/en.js (hunk 1 above)
Comment (Ssravani272) [reply deployment_specific/pb/ds_locales/en.js]: Fixed the issue

**Review by ruthvik129 — APPROVED** [review]

**Pull request merged**